tailwind.config.js

```diff
@@ -98,6 +98,7 @@ module.exports = {
         "fs-20": "20px",
         "fs-24": "24px",
         "fs-28": "28px",
+        "fs-48": "48px",
```

Comment:
max-w-[700px],leading-[60px],max-w-[860px] all this values can be defined here and used